lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1133,59 +1175,35 @@ public function get_stylesheet( $type = 'all' ) {
 	 * Merge new incoming data.
 	 *
 	 * @param WP_Theme_JSON $incoming Data to merge.
-	 * @param string        $origin origin of the incoming data (e.g: core, theme, or user).
 	 */
-	public function merge( $incoming, $origin ) {
-
+	public function merge( $incoming ) {
+		$origin           = $incoming->get_origin();
```

Comment:
Related to another comment I guess we don't need to know the $origin of data on the merge now, avoiding the call get_origin(). The data being merged can simply add new origins or overwrite if it comes from the same origin.

Reply:
Good catch! It was necessary for a previous implementation but it's no longer so removed it at https://github.com/WordPress/gutenberg/pull/32622/commits/20b8d044fafb738a85e801a5919274e549fcc542